action/store/postgres.py

```diff
@@ -86,8 +90,10 @@ def read(self) -> dict:
             table["fields"] = fields
             table_lookup[table_key] = table
 
+        metacopy = self.meta.copy()
+        metacopy.pop("excluded_tables", None)
         return {
-            **self.meta,
+            **metacopy,
             "tables": self._table_lookup_to_list(table_lookup),
```

Comment:
Since it's already a separate variable, just add the tables as

```py
meta["tables"] = self._table_lookup_to_list(table_lookup)
```

and return the `meta` variable. Otherwise, you're unnecessarily creating another dictionary.

Reply:
done